src/fancy-tools/wwwcRegionTool.js

```diff
@@ -2,102 +2,33 @@
 /* eslint no-underscore-dangle: 0 */
 import external from './../externalModules.js';
 import baseTool from './../base/baseTool.js';
-import EVENTS from './../events.js';
 // Drawing
 import { draw, drawRect, getNewContext } from '../util/drawing.js';
 import clip from '../util/clip.js';
 import getLuminance from '../util/getLuminance.js';
 import toolColors from './../stateManagement/toolColors.js';
 
-const isEmptyObject = (obj) => Object.keys(obj).length === 0 && obj.constructor === Object;
-
-/** Calculates the minimum, maximum, and mean value in the given pixel array */
-const calculateMinMaxMean = (storedPixelLuminanceData, globalMin, globalMax) => {
-  const numPixels = storedPixelLuminanceData.length;
-  let min = globalMax;
-  let max = globalMin;
-  let sum = 0;
-
-  if (numPixels < 2) {
-    return {
-      min,
-      max,
-      mean: (globalMin + globalMax) / 2
-    };
-  }
-
-  for (let index = 0; index < numPixels; index++) {
-    const spv = storedPixelLuminanceData[index];
-
-    min = Math.min(min, spv);
-    max = Math.max(max, spv);
-    sum += spv;
-  }
-
-  return {
-    min,
-    max,
-    mean: sum / numPixels
-  };
-};
-
-/** Calculates the minimum and maximum value in the given pixel array */
-const applyWWWCRegion = (evt, config) => {
-  const eventData = evt.detail;
-  const { image, element } = eventData;
-  const { start: startPoint, end: endPoint } = evt.detail.handles;
-
-  // Get the rectangular region defined by the handles
-  let left = Math.min(startPoint.x, endPoint.x);
-  let top = Math.min(startPoint.y, endPoint.y);
-  let width = Math.abs(startPoint.x - endPoint.x);
-  let height = Math.abs(startPoint.y - endPoint.y);
-
-  // Bound the rectangle so we don't get undefined pixels
-  left = clip(left, 0, image.width);
-  top = clip(top, 0, image.height);
-  width = Math.floor(Math.min(width, Math.abs(image.width - left)));
-  height = Math.floor(Math.min(height, Math.abs(image.height - top)));
-
-  // Get the pixel data in the rectangular region
-  const pixelLuminanceData = getLuminance(element, left, top, width, height);
-
-  // Calculate the minimum and maximum pixel values
-  const minMaxMean = calculateMinMaxMean(pixelLuminanceData, image.minPixelValue, image.maxPixelValue);
-
-  // Adjust the viewport window width and center based on the calculated values
-  const viewport = eventData.viewport;
-
-  if (config.minWindowWidth === undefined) {
-    config.minWindowWidth = 10;
-  }
-
-  viewport.voi.windowWidth = Math.max(Math.abs(minMaxMean.max - minMaxMean.min), config.minWindowWidth);
-  viewport.voi.windowCenter = minMaxMean.mean;
-
-  external.cornerstone.setViewport(element, viewport);
-  external.cornerstone.updateImage(element);
-};
-
 export default class extends baseTool {
   constructor (name) {
-    const strategies = {
-      applyWWWCRegion
-    };
-
     super({
       name: name || 'wwwcRegion',
```

Comment:
@swederik  gave me feedback on a previous PR that I think we should follow everywhere: replace
```name: name || 'wwwcRegion',```
with
```name,```
and then use a default parameter in the constructor:
```constructor (name = 'wwwcRegion) ```.